CHANGELOG.md

```diff
@@ -88,14 +88,21 @@ Adding a new version? You'll need three changes:
 
 ### Highlights
 
-- 🔒 Kong Gateway's [secret vaults](kong-vault) now become a first-class citizen for Kubernetes users
-  thanks to the new `KongVault` CRD.
-- 🔒 Providing an Enterprise license to KIC-managed Kong Gateways becomes much easier thanks to the new `KongLicense` CRD
-  which is used to dynamically provision all the replicas with the latest license found in the cluster.
-- ✨ Populating a single field of `KongPlugin`'s configuration with use of a Kubernetes Secret becomes possible thanks 
-  to the new `KongPlugin`'s `configPatches` field.
+- 🔒 Kong Gateway's [secret vaults](kong-vault) now become a first-class citizen for Kubernetes users thanks to the new
```

Comment:
These are dead link redirecting to e.g. https://github.com/Kong/kubernetes-ingress-controller/blob/546ea10fd2bbdfbaa67bd02fb7e15086172579ef/kong-vault 😅 

To make this work you'd need to use `[Nice text][link-ref-name]`

```suggestion
- 🔒 Kong Gateway's [secret vaults][kong-vault] now become a first-class citizen for Kubernetes users thanks to the new
```

This will have to be applied to several links in this section.

ref:https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Reply:
Oh, thanks for catching this 😅